worker/src/RTC/PortManager.cpp

```diff
@@ -217,6 +217,14 @@ namespace RTC
 			{
 				case Transport::UDP:
 				{
+#if defined(__linux__)
+					if (enableMulticast && Utils::IP::IsMulticast(&bindAddr, family))
+					{
+						flags |= UV_UDP_REUSEADDR;
+						MS_DEBUG_DEV("enabling UV_UDP_REUSEADDR");
```

Comment:
Logs must be placed first and blank line required.

```suggestion
					{
						MS_DEBUG_DEV("enabling UV_UDP_REUSEADDR");

						flags |= UV_UDP_REUSEADDR;
```